cypress/e2e/PepperDetails.cy.js

```diff
@@ -7,6 +7,22 @@ describe('All Peppers Flow', () => {
     })
   })
 
+  it('Should display an error to the user if pepper data does not load', () => {
+    cy.intercept('GET', 'https://polar-inlet-62371.herokuapp.com/peppers', {
+      statusCode: 400
+    })
+    cy.visit('http://localhost:3000/')
+    .contains('.error-message', 'Sorry, there has been a problem loading your page. Try again!')
+  })
+
+  it('Should display an error to the user if pepper data does not load', () => {
+    cy.intercept('GET', 'https://polar-inlet-62371.herokuapp.com/peppers', {
+      statusCode: 500
+    })
+    cy.visit('http://localhost:3000/')
+    .contains('.error-message', 'Sorry, there has been a problem loading your page. Try again!')
+  })
+
   it('Should be able to display the pepper name and picture', () => {
```

Comment:
I noticed when testing img visibility, when I broke a link and the broken link icon appears - it would still pass as visible, one suggestion I found google searching is to see if the image has a certain height(if they all have the same height) like this, 
 cy.get('img')
          .should('be.visible')
          .should(([img]) => {
            expect(img.height).to.equal(200)
          })